fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -2694,6 +2694,9 @@ public static boolean isNotCloudMode() {
     @ConfField(mutable = true, masterOnly = true)
     public static boolean enable_create_bitmap_index_as_inverted_index = false;
 
+    @ConfField(mutable = true)
+    public static boolean enable_create_inverted_index_for_array = false;
```

Comment:
If array_contains is OK, keep it true and disable index def with parser

Reply:
maybe we also need it for 2.1 because here depends on this pr: https://github.com/apache/doris/pull/32620 enable expr push down storage layer to pass inverted index , but this feature is not gonna to be in 2.1